doc/contributor-how-to/how-to-dev-plugin.md

```diff
@@ -0,0 +1,64 @@
+# How to install QGreenland Custom as a developer
+
+QGreenland Custom is a QGIS plugin for downloading a custom set of data, including data which 
+is not part of the QGreenland Core zip package. This how-to guide walks through how to install the plugin as a developer.
+
+First, clone the QGreenland Custom repo from Faunalia:
+
+```
+git clone https://github.com/faunalia/qgreenland-plugin.git
+```
+
+Use the QGIS GUI to locate the QGreenland Custom directory by going to
+Settings -> User Profiles -> Open Active Profile Folder.
+Then, navigate to the /python/plugins directory, PWD and copy the full filpath. 
+
+Next symlink the plugin repo to the plugin directory:
+```
+ln -s $PWD '/filepath'
```

Comment:
It's not clear here that `/filepath` should be replaced with the path that was copied in the previous step. Do we have similar instructions elsewhere in the documentation? If there is anything pre-existing, we should try and be consistent with that; otherwise, we have the harder job of establishing a clear style :)